modules/protect/math-fallback.php

```diff
@@ -124,11 +124,19 @@ static function math_form() {
 			?>
 			<div style="margin: 5px 0 20px;">
 				<strong><?php esc_html_e( 'Prove your humanity:', 'jetpack' ); ?> </strong>
-				<?php echo $num1 ?> &nbsp; + &nbsp; <?php echo $num2 ?> &nbsp; = &nbsp;
-				<input type="input" name="jetpack_protect_num" value="" size="2" />
-				<input type="hidden" name="jetpack_protect_answer" value="<?php echo $ans; ?>" />
+				<label for="sum">
+					<?php echo esc_html( $num1 ); ?>
```

Comment:
If you look at line 120 in this file, you have
```php
$num1 = rand( 0, 10 );
$num2 = rand( 1, 10 );
...
$ans  = sha1( $salt . $sum );
```
so `esc_html` is not really needed here, or in the line outputting `$num2` or in the line outputting `$ans` as an attribute. Escaping should be performed when the input is provided by the user, which includes the case of translations, where `esc_html_e` is now properly used because translation are also a source of vulnerabilities. However, these values are generated here by code without user input so it's safe to remove the escaping.
After this is addressed, @oskosk or @simison can give a final test and approve for merge if they consider it final.

Reply:
@eliorivero 
I initially didn't, but if you don't, the CodeSnifffer complains.

"All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbook)"
https://developer.wordpress.org/plugins/security/securing-output/#escaping

Since I already have them escaped, do you absolutely want this to be removed at this point?

Is there a second opinion on this? It's not harmful to have escaping and it also solves the warning.